utils/http.ts

```diff
@@ -16,3 +17,16 @@ export function respondJSON(
     },
   };
 }
+
+export function parseRequestBody(
+  event: APIGatewayProxyEventV2,
+): APIGatewayProxyEventV2 {
+  const headers = new Headers(event.headers);
+  if (
+    headers.get("content-type") === "application/x-www-form-urlencoded" &&
+    event.body
+  ) {
+    event.body = atob(event.body);
```

Comment:
1. I think this base64 decode should happen based on the `isBase64Encoded` property on the `APIGatewayProxyEventV2`, not based on `content-type`. This base64 comes from AWS, not because the content is `x-www-form-urlencoded`. 

2. GitHub doesn't just send JSON when using `x-www-form-urlencoded`. You need to strip the `payload=` from the start of the body, and run it thru `decodeURIComponent` to get the JSON body.